Travis-CI is returning error on Python 3.7

Hi,

Here is the output: https://app.travis-ci.com/github/kytos-ng/kytos/jobs/529070298

```
============================= test session starts ==============================
567platform linux -- Python 3.7.1, pytest-5.4.1, py-1.10.0, pluggy-0.13.1
568rootdir: /home/travis/build/kytos-ng/kytos, inifile: setup.cfg
569collected 320 items                                                            
570
571tests/unit/test_core/test_api_server.py ................................ [ 10%]
572......                                                                   [ 11%]
573tests/unit/test_core/test_atcp_server.py ..FF                            [ 13%]
574tests/unit/test_core/test_auth.py ..........                             [ 16%]
575tests/unit/test_core/test_buffers.py ..........                          [ 19%]
576tests/unit/test_core/test_common.py .....................                [ 25%]
577tests/unit/test_core/test_connection.py ...............                  [ 30%]
578tests/unit/test_core/test_controller.py ................................ [ 40%]
579......................                                                   [ 47%]
580tests/unit/test_core/test_events.py .....                                [ 49%]
581tests/unit/test_core/test_exceptions.py .....                            [ 50%]
582tests/unit/test_core/test_helpers.py ....                                [ 51%]
583tests/unit/test_core/test_interface.py ................................. [ 62%]
584.                                                                        [ 62%]
585tests/unit/test_core/test_kytosd.py ......                               [ 64%]
586tests/unit/test_core/test_link.py .................                      [ 69%]
587tests/unit/test_core/test_logs.py .............                          [ 73%]
588tests/unit/test_core/test_napp_dir_listener.py ....                      [ 75%]
589tests/unit/test_core/test_napps_base.py ..................               [ 80%]
590tests/unit/test_core/test_napps_manager.py .......................       [ 87%]
591tests/unit/test_core/test_switch.py ...............................      [ 97%]
592tests/unit/test_core/test_websocket.py .                                 [ 97%]
593tests/unit/test_lib/test_helpers.py .......                              [100%]
594
595=================================== FAILURES ===================================
596__________________ TestKytosServerProtocol.test_data_received __________________
597
598self = <tests.unit.test_core.test_atcp_server.TestKytosServerProtocol object at 0x7f17e778e898>
599mock_kytos_event = <MagicMock name='KytosEvent' id='139740644423832'>
600
601    @patch('kytos.core.atcp_server.KytosEvent')
602    def test_data_received(self, mock_kytos_event):
603        """Test data_received method."""
604        buffers = self.server_protocol.server.controller.buffers
605        self.connection.protocol.name = 'protocol'
606    
607>       self.server_protocol.data_received(b'data')
608
609tests/unit/test_core/test_atcp_server.py:92: 
610_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
611kytos/core/atcp_server.py:173: in data_received
612    self._loop.create_task(self.server.controller.buffers.raw.aput(event))
613_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
614
615self = <_UnixSelectorEventLoop running=False closed=False debug=False>
616coro = <MagicMock name='mock.controller.buffers.raw.aput()' id='139740644122128'>
617
618    def create_task(self, coro):
619        """Schedule a coroutine object.
620    
621        Return a task object.
622        """
623        self._check_closed()
624        if self._task_factory is None:
625>           task = tasks.Task(coro, loop=self)
626E           TypeError: a coroutine was expected, got <MagicMock name='mock.controller.buffers.raw.aput()' id='139740644122128'>
627
628/opt/python/3.7.1/lib/python3.7/asyncio/base_events.py:394: TypeError
629_________________ TestKytosServerProtocol.test_connection_lost _________________
630
631self = <tests.unit.test_core.test_atcp_server.TestKytosServerProtocol object at 0x7f17e7784f98>
632mock_kytos_event = <MagicMock name='KytosEvent' id='139740644039480'>
633
634    @patch('kytos.core.atcp_server.KytosEvent')
635    def test_connection_lost(self, mock_kytos_event):
636        """Test connection_lost method."""
637        buffers = self.server_protocol.server.controller.buffers
638        self.connection.protocol.name = 'protocol'
639    
640>       self.server_protocol.connection_lost('exc')
641
642tests/unit/test_core/test_atcp_server.py:106: 
643_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
644kytos/core/atcp_server.py:194: in connection_lost
645    self._loop.create_task(self.server.controller.buffers.app.aput(event))
646_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
647
648self = <_UnixSelectorEventLoop running=False closed=False debug=False>
649coro = <MagicMock name='mock.controller.buffers.app.aput()' id='139740643535112'>
650
651    def create_task(self, coro):
652        """Schedule a coroutine object.
653    
654        Return a task object.
655        """
656        self._check_closed()
657        if self._task_factory is None:
658>           task = tasks.Task(coro, loop=self)
659E           TypeError: a coroutine was expected, got <MagicMock name='mock.controller.buffers.app.aput()' id='139740643535112'>
660
661/opt/python/3.7.1/lib/python3.7/asyncio/base_events.py:394: TypeError
662=============================== warnings summary ===============================
663tests/unit/test_core/test_auth.py: 10 tests with warnings
664tests/unit/test_core/test_common.py: 21 tests with warnings
665tests/unit/test_core/test_controller.py: 54 tests with warnings
666tests/unit/test_core/test_interface.py: 29 tests with warnings
667tests/unit/test_core/test_link.py: 17 tests with warnings
668tests/unit/test_core/test_napps_manager.py: 23 tests with warnings
669tests/unit/test_core/test_switch.py: 31 tests with warnings
670tests/unit/test_lib/test_helpers.py: 2 tests with warnings
671  /home/travis/build/kytos-ng/kytos/kytos/core/config.py:167: UserWarning: Unknown arguments: ['tests/unit']
672    warnings.warn(f"Unknown arguments: {unknown}")
673
674tests/unit/test_core/test_auth.py: 10 tests with warnings
675  /home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jwt/api_jwt.py:81: DeprecationWarning: It is strongly recommended that you pass in a value for the "algorithms" argument when calling decode(). This argument will be mandatory in a future version.
676    DeprecationWarning
677
678tests/unit/test_core/test_auth.py::TestAuth::test_02_list_users_request
679tests/unit/test_core/test_auth.py::TestAuth::test_03_create_user_request
680tests/unit/test_core/test_auth.py::TestAuth::test_03_create_user_request_error
681tests/unit/test_core/test_auth.py::TestAuth::test_04_list_user_request
682tests/unit/test_core/test_auth.py::TestAuth::test_04_list_user_request_error
683tests/unit/test_core/test_auth.py::TestAuth::test_05_update_user_request
684tests/unit/test_core/test_auth.py::TestAuth::test_05_update_user_request_error
685tests/unit/test_core/test_auth.py::TestAuth::test_06_delete_user_request
686tests/unit/test_core/test_auth.py::TestAuth::test_06_delete_user_request_error
687  /home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jwt/api_jws.py:145: DeprecationWarning: It is strongly recommended that you pass in a value for the "algorithms" argument when calling decode(). This argument will be mandatory in a future version.
688    DeprecationWarning
689
690tests/unit/test_core/test_connection.py::TestConnection::test_state__error
691  /home/travis/build/kytos-ng/kytos/kytos/core/connection.py:67: DeprecationWarning: using non-Enums in containment checks will raise TypeError in Python 3.8
692    if new_state not in ConnectionState:
693
694tests/unit/test_core/test_kytosd.py::TestKytosd::test_main__foreground
695  /opt/python/3.7.1/lib/python3.7/unittest/mock.py:1865: RuntimeWarning: coroutine 'async_main.<locals>.start_shell_async' was never awaited
696    setattr(_type, entry, MagicProxy(entry, self))
697
698-- Docs: https://docs.pytest.org/en/latest/warnings.html
699=========================== short test summary info ============================
700
```

--- a/tests/unit/test_core/test_atcp_server.py
+++ b/tests/unit/test_core/test_atcp_server.py
@@ -11,6 +11,7 @@ from kytos.core.atcp_server import (KytosServer, KytosServerProtocol,
 # Using "nettest" TCP port as a way to avoid conflict with a running
 # Kytos server on 6653.
 TEST_ADDRESS = ('127.0.0.1', 4138)
+# pylint: disable=protected-access
 
 
 class TestKytosServer:

--- a/tests/unit/test_core/test_atcp_server.py
+++ b/tests/unit/test_core/test_atcp_server.py
@@ -80,6 +80,7 @@ class TestKytosServerProtocol:
         self.connection.port = 123
 
         self.server_protocol = KytosServerProtocol()
+        self.server_protocol._loop = MagicMock()
         self.server_protocol.server = MagicMock()
         self.server_protocol.connection = self.connection
 

